Consul.Test/OperatorTest.cs

```diff
@@ -104,5 +104,11 @@ public async Task Operator_CreateArea()
             var response = await _client.Operator.CreateArea(check);
             Assert.NotNull(response.Response);
         }
+        [EnterpriseOnlyFact]
+        public async Task Operator_AreaList()
+        {
+            var req = await _client.Operator.AreaList();
+            Assert.NotNull(req.Response);
```

Comment:
To make this test more exhaustive, it would be nice to create an area first and then assert that the query result is non-empty (instead of not null).

Reply:
Thanks, done ✅

Reply:
Pleas also updated the test condition, from not null to non-empty. Please inspect also a result fields if they are filled in.